numpy/core/src/multiarray/compiled_base.dispatch.c

```diff
@@ -0,0 +1,94 @@
+/**
+ * @targets $maxopt baseline
+ * SSE2 AVX2 AVX512F
+ * VSX VSX2
+ * NEON ASIMDDP
+ */
+#include "compiled_base_pack_inner.h"
```

Comment:
```suggestion
#include "compiled_base_pack_inner.h"
#include <string.h>
#include "simd/simd.h"
#include "lowlevel_strided_loops.h" /* for npy_bswap8 */
```
moved from that header